rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1082,84 +1080,83 @@ private void processReleaseRequests(Transaction rskTx) {
         );
 
         if (activations.isActive(RSKIP271)) {
-            processPegoutsInBatch(releaseRequestQueue, txBuilder, availableUTXOs, releaseTransactionSet, activeFederationWallet, rskTx);
+            processPegoutsInBatch(pegoutRequests, txBuilder, availableUTXOs, pegoutsWaitingForConfirmations, activeFederationWallet, rskTx);
         } else {
-            processPegoutsIndividually(releaseRequestQueue, txBuilder, availableUTXOs, releaseTransactionSet, activeFederationWallet);
+            processPegoutsIndividually(pegoutRequests, txBuilder, availableUTXOs, pegoutsWaitingForConfirmations, activeFederationWallet);
         }
     }
 
-    private void addPegoutTxToReleaseTransactionSet(
+    private void addPegoutBtcTxToPegoutWaitingForConfirmationsSet(
```

Comment:
```suggestion
    private void addToPegoutsWaitingForConfirmations(
```
A bit shorter, what do you think?